views/components/collapsibleBlock.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * All About Olaf
+ * Individual row for inside a collapsible view
+ */
+
+import React from 'react'
+import {
+  View,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native'
+
+import Collapsible from 'react-native-collapsible'
+
+
+export default class CollapsibleBlock extends React.Component {
+
+  static propTypes = {
+    backgroundColor: React.PropTypes.string.isRequired,
+    children: React.PropTypes.array.isRequired,
+  }
+
+  state = {
+    isCollapsed: true,
+  }
+
+  styles = StyleSheet.create({
+    container: {
+      marginBottom: 0,
+    },
+    collapsedContent: {
+      alignSelf: 'center',
+
+    },
+    collapsedContainer: {
+    },
+  })
+
+  render() {
+    return (
+        <TouchableOpacity
+          onPress={() => this.setState({isCollapsed: !this.state.isCollapsed})}
+        >
+          {this.props.children[0]}
```

Comment:
Um… Let me think about this and get back to you. It's generally a bad idea to do anything to `props.children`, as it won't be an array if there's only one child… which there shouldn't be, but … hmm. Referencing https://facebook.github.io/react/tips/children-props-type.html…

It looks like we should use the React.children.toArray helper function to ensure that this is an array: https://facebook.github.io/react/docs/top-level-api.html#react.children.toarray